packages/@lwc/babel-plugin-component/src/decorators/index.ts

```diff
@@ -142,29 +152,32 @@ function generateInvalidDecoratorError(path) {
     }
 }
 
-function collectDecoratorPaths(bodyItems) {
-    return bodyItems.reduce((acc, bodyItem) => {
+function collectDecoratorPaths(bodyItems: NodePath<types.Node>[]): NodePath<types.Decorator>[] {
+    return bodyItems.reduce((acc: NodePath<types.Decorator>[], bodyItem) => {
         const decorators = bodyItem.get('decorators');
-        if (decorators && decorators.length) {
-            acc.push(...decorators);
+        if (Array.isArray(decorators)) {
```

Comment:
This looks like a functional change. Before, the condition would evaluate to `false` if `decorators === []`. Now, it'll evaluate to `true`.

Reply:
Good catch! You're right, I got lazy with this one. Was trying to get TypeScript to realize that this is an array. Fixed.